codegenerator/cli/templates/dynamic/codegen/src/RegisterHandlers.res.hbs

```diff
@@ -145,18 +145,13 @@ let registerContractHandlers = (
                 {{#each chain_config.codegen_contracts as | contract |}}
                 {
                   name: "{{contract.name.capitalized}}",
-                  addresses: [
-                    {{#each contract.addresses as | address |}}
-                    "{{address}}"->Address.unsafeFromString,
-                    {{/each}}
-                  ],
```

Comment:
Are these static addresses never used?

Reply:
Yes. They are taken from contractConfig outside of the HyperFuelWorker